### PR TITLE
DCOS-44661: extract data-service package into own repository

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
## Testing

Check if the jobs and repositories page behave the same way as before

## Trade-offs

- Decided to do also an auto-update for this one

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -209,7 +210,7 @@
     "serve-notify": "export NOTIFY=true && npm run serve",
     "scaffold": "./node_modules/.bin/gulp ensureConfig && ./node_modules/.bin/gulp ensureDevProxy",
     "prestart": "npm run check-env",
-    "start": "lingui compile && NODE_ENV=development webpack-dev-server --config ./webpack/webpack.dev.js --hot --progress --colors --content-base ./build --host 0.0.0.0 --port $npm_package_config_port",
+    "start": "lingui compile && NODE_ENV=development webpack-dev-server --config ./webpack/webpack.dev.js --display-error-details --hot --progress --colors --content-base ./build --host 0.0.0.0 --port $npm_package_config_port",
```
Comment (nLight) [inline package.json]: Something is being snuck in 🕵️

File: renovate.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
         "@dcos/http-service",
         "@dcos/mesos-client",
         "@dcos/tslint-config",
-        "@dcos/ui-kit"
+        "@dcos/ui-kit",
+        "@dcos/data-service"
```
Comment (nLight) [inline renovate.json]: 💜

**Comment by nLight** [comment]
please update l10n

**Comment by DanielMSchmidt** [comment]
@nLight done

**Pull request merged**